### ISSUE TITLE
Catatonia

### ISSUE THREAD

**Issue by Tyler-Dietterich** [obophenotype/human-phenotype-ontology#10535]
**Preferred term label:**
Catatonia

**Synonyms**
Catalepsy

**Definition (free text, please give PubMed ID)**
Neuropsychiatric syndrome associated with severe mental illness, commonly involving abnormal movements and behaviors such as stupor, catalepsy, waxy flexibility, mutism, negativism, posturing, mannerisms, stereotypy, and agitation. 
PubMed ID: 28613592

**Parent term (use hpo.jax.org/app)**
Abnormal nervous system physiology, HP:0012638

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Schizophrenia, OMIM:181500 (https://omim.org/entry/181500)
Schizophrenia 10, OMIM: [605419](https://www.omim.org/entry/605419)
Epilepsy, nocturnal frontal lobe, 5, OMIM: [615005](https://omim.org/entry/615005)
Kleefstra syndrome, OMIM:610253 (https://omim.org/entry/610253)
Peripheral neuropathy with variable spasticity, exercise intolerance, and developmental delay, OMIM:616539 (https://omim.org/entry/616539)
Peripheral neuropathy, autosomal recessive, with or without impaired intellectual development, OMIM:618124 (https://omim.org/entry/618124)
Neuroleptic malignant syndrome, ORPHA:94093 (https://www.orpha.net/consor/cgi-bin/OC_Exp.php?Lng=EN&Expert=94093)
NMDA receptor encephalitis, ORPHA:217253 (https://www.orpha.net/consor/cgi-bin/OC_Exp.php?Lng=EN&Expert=217253)

**Your nano-attribution (ORCID)**
0000-0003-1909-0010

**Comment by pnrobinson** [comment]
@bcoleman96 
I think that catatonia is not an observable phenotypic feature but rather a description of a variable combination of anomalies. Probably the HPO would want to have features for each of the specific symptoms, rather than group them together like this?

**Comment by pnrobinson** [comment]
@Tyler-Dietterich 
I think this is more of a disease entity than a phenotypic feature. 
As such you could use the MONDO, https://www.ebi.ac.uk/ols4/ontologies/mondo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FMONDO_0800105
